internal/proxy/task_query.go

```diff
@@ -127,15 +128,24 @@ func filterSystemFields(outputFieldIDs []UniqueID) []UniqueID {
 // parseQueryParams get limit and offset from queryParamsPair, both are optional.
 func parseQueryParams(queryParamsPair []*commonpb.KeyValuePair) (*queryParams, error) {
 	var (
-		limit  int64
-		offset int64
-		err    error
```

Comment:
milvus has an underlying constraint that if there's no limit set, then no considering offset. But reduceStopForBest is independent from limit or offset